core/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -972,7 +974,7 @@ public void testRecoverFromStore() throws IOException {
         IndexService test = indicesService.indexService(resolveIndex("test"));
         final IndexShard shard = test.getShardOrNull(0);
         int translogOps = 1;
-        client().prepareIndex("test", "test", "0").setSource("{}").setRefresh(randomBoolean()).get();
+        client().prepareIndex("test", "test", "0").setSource("{}").setRefreshPolicy(randomFrom(RefreshPolicy.values())).get();
```

Comment:
Yeah, I think this was the wrong choice. I wanted to keep the spirit of the old test and just do a random thing, but I don't think it is worth it. Just random picking between IMMEDIATE and NONE is probably more right.